.github/workflows/images.yaml

```diff
@@ -37,6 +37,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
Ah, that makes sense. I recall that checkout actioni used `fetch-depth: 1` by default and doesn't fetch tags? 🤔

Reply:
yes. I debug this with long time.